src/test/java/com/github/dockerjava/core/command/CreateContainerCmdImplTest.java

```diff
@@ -735,4 +737,20 @@ public void createContainerWithCgroupParent() throws DockerException {
 
         assertThat(inspectContainer.getHostConfig().getCgroupParent(), is("/parent"));
     }
+
+    @SuppressWarnings("Duplicates")
```

Comment:
What does it mean? If it about code duplication, then keep it as i still wonder to do parametrized tests :(